### PR TITLE
Add error code explanation for E0224

### PR DESCRIPTION
**Pull request by chansuke**
Part of #61137

### CODE REVIEW

File: src/librustc_error_codes/error_codes/E0224.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+This fails because you are using dynamically sized trait objects without Trait
```
Comment (Dylan-DPC) [inline src/librustc_error_codes/error_codes/E0224.md]:
```suggestion
This fails because you are using dynamically sized trait objects without specifying a trait
```

File: src/librustc_error_codes/error_codes/E0224.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+Erroneous code example:
+
+```compile_fail,E0224
+fn main() {
+    let _x: &dyn = &0;
+}
+```
+
+To solve the error please add at least one trait for `&dyn`
```
Comment (Dylan-DPC) [inline src/librustc_error_codes/error_codes/E0224.md]:
```suggestion
To solve this error, please add at least one trait for `&dyn`
```

**Review by Dylan-DPC-zz — CHANGES_REQUESTED** [review]

File: src/librustc_error_codes/error_codes/E0224.md (hunk 1 above)
Comment (GuillaumeGomez) [reply src/librustc_error_codes/error_codes/E0224.md]: I'd reword it to (integrating @Dylan-DPC suggestion): "A dynamically sized trait was used without specifying a trait".

**Comment by JohnCSimon** [comment]
Ping from triage: 
@chansuke Can you please address the test failures and the change request from dylan-DPC?

**Comment by JohnCSimon** [comment]
Pinging again from triage:
@chansuke - can you please post the status of this PR?

**Comment by joelpalmer** [comment]
Ping from Triage: Any updates @chansuke?

**Comment by Dylan-DPC** [comment]
Hi @chansuke. I'm closing this due to inactivity. Feel free to reoopen it and ping me. Thanks
